src/modules/pokemons/PokemonList.ts

```diff
@@ -14062,7 +14062,6 @@ export const pokemonList = createPokemonArray(
         'levelType': LevelType.slow,
         'exp': 78,
         'catchRate': 50,
-        'evolutions': [StoneEvolution('Elf Munchlax', 'Santa Snorlax', StoneType.Soothe_bell)],
```

Comment:
Can we then hardcode the computed 32 EC for Elf Munchlax until it gets killed too? Its eggsteps were going down from 1600 to 1280 thanks to baby evolution.